### ISSUE TITLE
Migration does not work

### ISSUE THREAD

**Issue by a0g83agbc84** [pardom-zz/ActiveAndroid#199]
I have an app published with AA_DB_VERSION = 1.
This is my v1 Model:

```
@SuppressWarnings("serial")
@Table(name = "Blog")
public class Portal extends Model implements Serializable {

    @Column(name="title")
    private String mTitle;

    @Column(name="description")
    private String mDescription;

    @Column(name="url")
    private String mUrl;

    @Column(name="color")
    private int mColor;

    public Portal() {
        super();
    }

    public Portal(final String title, final String description, final String url, final int color,
            final String category, final String country) {
        setTitle(title);
        setDescription(description);
        setUrl(url);
        setColor(color);
    }

    public String getTitle() {
        return mTitle;
    }

    public void setTitle(final String title) {
        mTitle = title;
    }

    public String getDescription() {
        return mDescription;
    }

    public void setDescription(final String description) {
        mDescription = description;
    }

    public String getUrl() {
        return mUrl;
    }

    public void setUrl(final String url) {
        mUrl = url;
    }

    public int getColor() {
        return mColor;
    }

    public void setColor(final int color) {
        mColor = color;
    }
}
```

Then, I added two new columns:

```
@Column(name="category")
private String mCategory;

@Column(name="country")
private String mCountry;
```

I bumped the AA_DB_VERSION to 2 and added a folder named 'migrations' inside the 'assets' folder. And added a '2.sql' file with the following content:

```
ALTER TABLE Blog ADD COLUMN category TEXT;
ALTER TABLE Blog ADD COLUMN country TEXT;
```

After updating the app, I get the following errors:

`03-23 12:31:29.566: E/SQLiteLog(7531): (1) table Blog has no column named country`

Isn't this the way to update the table?

Thank you!

**Comment by matteinn** [comment]
Hi!
I have faced the same issue and I found out that you can only write one command per migration file. So, in your situation, you have to bump the db version to 3 and use 2.sql and 3.sql each with one alter table command.
This worked for me.
Bye

**Comment by a0g83agbc84** [comment]
Thank you matteinn, but... I think this is a bit confusing way... Isn't it easier to place in one file all the migratino? I mean, I just added two columns but, if in the future, I need to add a lot of new stuff to that table? Do I need to create n sql files? Doesn't sound good...

Can't the owner of this project fix and create a new JAR? I think it'll solve many problems...

**Comment by joshuapinter** [comment]
@KrLx1994roller @Vonloxley has already issued a pull request that will allow multiple SQL commands in a single migration file. It just needs to get reviewed by @SeanPONeil and merged in to allow this functionality.

**Comment by a0g83agbc84** [comment]
@joshuapinter Ok! Thank you! Glad to help :+1:

**Comment by joshuapinter** [comment]
You bet. That's a good feature, thanks for pointing it out.

**Comment by SeanPONeil** [comment]
merged #200
